### DISCUSSION TITLE
Plex v5 Script - No Hardware Encoding

### DISCUSSION THREAD

**Discussion by patchrick84** [tteck/Proxmox#932] [Help Needed]
Hi there!

Forgive my relative newbness, I'm slowly learning my way around Proxmox and came across this script.

I have an HP Mini Elite 600 G9 - which has an Intel i7-12700T.  I've tried searching all over the Googles but can't find a solution to my problem.

If I run the script and install using 20.04, all goes well except the checkbox for hardware transcoding doesn't even show up in Plex.  If I run the script using the newer Ubuntu versions (which I tried thinking the newer machine might need a newer kernel), it fails with the unknown error.

I've followed other threads/guides I've found that say to install Intel drivers, edit the conf files (which was already done by the script anyway) and more, but nothing fixes it.  I'm not sure how to proceed to get it going, so any guidance would be greatly appreciated!

**Comment by tteck** [comment]
https://github.com/tteck/Proxmox/discussions/179#discussioncomment-4634325  🤷🏻‍♂️

**Comment by patchrick84** [comment]
I'm an idiot.  Forgot this was a Plex Pass only feature and hadn't "claimed" my server under my account.  There goes a whole day's work for nothing!!  🤦🏻‍♂️

Thanks for the suggestion, @tteck!  Please don't kick me out for this infraction!  😂
